src/synapse/azext_synapse/_help.py

```diff
@@ -230,17 +285,16 @@
 helps['synapse spark batch create'] = """
 type: command
 short-summary: Submit spark batch job.

```

Comment:
>spark [](start = 22, length = 5)

Spark. Same with other occurrences.

Reply:
fixed

---
In reply to: [417311302](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r417311302) [](ancestors = 417311302)